website/react-magma-docs/src/pages/api-intro/page-templates.mdx

```diff
@@ -278,17 +365,88 @@ export function Example() {
   );
 
   return (
-
-      <Container>
-
-        { isLargeScreen ? <Drawer position={DrawerPosition.left} isOpen={openDrawer}><Sidebar/></Drawer> : <Sidebar/> }
-
-        <StyledHeader isInverse logo={<strong>LOGO</strong>} breakpoint={magma.breakpoints.large} isCompact={isSmallerScreen} onMenuButtonClick={() => {setOpenDrawer(!openDrawer)}}/>
-
-        <Content>Content</Content>
-
-      </Container>
-
+    <Grid
+      gridJustifyContent={GridJustifyContent.center}
+      gridTemplateColumns="auto 1fr"
+      gridTemplateRows="auto 1fr"
+      gridTemplateAreas={('header header', 'sidebar content', 'sidebar footer')}
+      gridGap="2px"
+    >
+      <GridItem as="header" gridColumn="1 / 4">
+        <Header
+          isInverse
+          logo={<strong>LOGO</strong>}
+          breakpoint={magma.breakpoints.large}
+          isCompact={isSmallerScreen}
+          onMenuButtonClick={() => {
+            setOpenDrawer(!openDrawer);
+          }}
+        />
+      </GridItem>
+
+      {isLargeScreen ? (
+        <Drawer position={DrawerPosition.left} isOpen={openDrawer}>
+          <div
+            style={{
+              background: magma.colors.primary200,
+              padding: magma.spaceScale.spacing05,
+              width: '240px',
+              height: 'calc(100vh - 32px)',
+            }}
+          >
+            Sidebar
+          </div>
+        </Drawer>
+      ) : (
+        <GridItem
+          as="sidebar"
```

Comment:
Is `as` equivalent to `gridArea`?

Reply:
It should but they do not behave the same hence why I kept it with the as per the documentation examples.